payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -1003,7 +1003,13 @@ constructor(
          *
          * [us_bank_account.linkedAccount](https://stripe.com/docs/api/payment_methods/object#payment_method_object-us_bank_account-linked_account)
          */
-        @JvmField val linkedAccount: String?,
+        @Deprecated(
+            message = "Renamed to 'financialConnectionsAccount', " +
+                "'linkedAccount' will be removed in a future major update",
+            replaceWith = ReplaceWith(expression = "financialConnectionsAccount")
+        )
+        @JvmField
+        val linkedAccount: String?,
```

Comment:
Can we just rename this field now and provide `linkedAccount` as a deprecated computed property?